### PR TITLE
Adding touchstone export from siwave exec file

### PR DESCRIPTION
**Pull request by svandenb-dev**
Added the touchstone file export with siwave exec file creation

### CODE REVIEW

File: pyaedt/edb_core/siwave.py [hunk 1]
```diff
@@ -742,8 +742,24 @@ def create_dc_terminal(
...
+        ----------
+        add_dc : bool
+            Add the DC option in the exec file
+        add_ac : bool
+            Add the AC option in the exec file.
+        add_syz : bool
+            Add the SYZ option in the exec file
+        export_touchstone : bool
+            Add the Touchstone file export option in the exec file
+        touchstone_file_path : str
```
Comment (MaxJPRey) [inline pyaedt/edb_core/siwave.py:759]:
```suggestion
        add_dc : bool, optional
            Add the DC option in the exec file
        add_ac : bool, optional
            Add the AC option in the exec file.
        add_syz : bool, optional
            Add the SYZ option in the exec file
        export_touchstone : bool, optional
            Add the Touchstone file export option in the exec file
        touchstone_file_path : str, optional
```

File: pyaedt/edb_core/siwave.py [hunk 2]
```diff
@@ -742,8 +742,24 @@ def create_dc_terminal(
...
+            When string is empty and ``export_touchstone`` == ``True``, The default file path will be at the project
```
Comment (MaxJPRey) [inline pyaedt/edb_core/siwave.py:760]:
```suggestion
            When string is empty and ``export_touchstone`` == ``True``, the default file path will be at the project
```

**Review by MaxJPRey — APPROVED** [review]
LGTM.

File: pyaedt/edb_core/siwave.py [hunk 3]
```diff
@@ -740,8 +740,24 @@ def create_dc_terminal(
         return self.create_pin_group_terminal(dc_source)
 
     @pyaedt_function_handler()
-    def create_exec_file(self, add_dc=False, add_ac=False, add_syz=False):
-        """Create an executable file."""
+    def create_exec_file(
+        self, add_dc=False, add_ac=False, add_syz=False, export_touchstone=False, touchstone_file_path=""
+    ):
+        """Create an executable file.
+        Parameters
```
Comment (PipKat) [inline pyaedt/edb_core/siwave.py:747]:
```suggestion
        
        Parameters
```
A blank line is needed before the Parameters section for proper rendering in the doc.

File: pyaedt/edb_core/siwave.py [hunk 4]
```diff
@@ -740,8 +740,24 @@ def create_dc_terminal(
...
+            Add the DC option in the exec file
+        add_ac : bool, optional
+            Add the AC option in the exec file.
+        add_syz : bool, optional
+            Add the SYZ option in the exec file
+        export_touchstone : bool, optional
+            Add the Touchstone file export option in the exec file
+        touchstone_file_path : str, optional
+            When string is empty and ``export_touchstone`` == ``True``, the default file path will be at the project
+            location.
```
Comment (PipKat) [inline pyaedt/edb_core/siwave.py:759]:
```suggestion
        add_dc : bool, optional
            Whether to add the DC option in the EXE file. The default is ``False``.
        add_ac : bool, optional
            Whether to add the AC option in the EXE file. The default is
            ``False``.
        add_syz : bool, optional
            Whether to add the SYZ option in the EXE file
        export_touchstone : bool, optional
            Add the Touchstone file export option in the EXE file.
            The default is ``False``.
        touchstone_file_path : str, optional
            File path for the Touchstone file. The default is ``""``.  When no path is
            specified and ``export_touchstone=True``, the path for the project is
            used.
```
I don't see `add_syz`` in the method signature.

**Pull request merged**